src/rust/python-parser/lib.rs

```diff
@@ -0,0 +1,40 @@
+use std::collections::HashSet;
+
+use ruff_python_ast::{Stmt, StmtImportFrom};
+use ruff_python_parser::parse_module;
+
+#[cxx::bridge(namespace = "edgeworker::rust::python_parser")]
+mod ffi {
+
```

Comment:
I don't think we have formatting enabled for Rust files. Can you run `just rustfmt` to see if any formatting changes are being made? If yes, we need to fix this (in a separate PR)

Reply:
we do and `--config=lint` should be checking it. Let me know if this is not the case.

Reply:
I ran this and don't see any changes. Maybe it's already set up correctly?